src/app/features/public-form/edit-public-form-columns/edit-public-form-columns.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="hint-banner" i18n>
+  You can edit how users will see the details of this public form. To edit
+  fields, click on the Edit button, make your changes, and then save. These
+  changes will reflect on the public form.
+  <br />
+  Drag and drop fields and sections in this preview of a profile view. The
+  editor below closely resembles how the form will look for users later.
+  <br />
+</div>
+<div [class.disabled]="formControl.disabled">
```

Comment:
can we move this class.disabled into the AdminEntityForm component and make it part of the logic for the isDisabled input? Then developers only need to toggle this with one single variable